pkgs/development/libraries/qt-4.x/4.8/default.nix

```diff
@@ -183,10 +183,36 @@ stdenv.mkDerivation rec {
     sed -i 's/^\(LIBS[[:space:]]*=.*$\)/\1 -lobjc/' ./src/corelib/Makefile.Release
   '';
 
-  postInstall =
-    ''
+  installPhase = optionalString stdenv.isDarwin ''
```

Comment:
There should be a `runHook preInstall` here.

Similarly for `postInstall` at the end.

Reply:
Fixed